collections/hub/sdk-bulk/bulk-happy-path.json

```diff
@@ -53,51 +54,52 @@
                 "to": {
                   "partyIdInfo": {
                     "partyIdType": "MSISDN",
-                    "partyIdentifier": "56781234001"
+                    "partyIdentifier": "{$inputs.SIM2_MSISDN_1}"
```

Comment:
Can we call this as TTKSIM1_MSISDN_1 instead? Because we are using SIM1, SIM2...etc already.
Please change SIM1 to TTKSIM1, SIM2 to TTKSIM2 in all the places

Reply:
I was hoping to leave it as generic name because different environments have different naming requirements for the sims. So if in a different env we had a requirement to name the sim as something different than TTKSIM, I felt leaving it as SIM is generic enough @vijayg10

Reply:
SIM1, SIM2 are already used for testfsp1, testfsp2...etc

Reply:
@vijayg10...took care of this